javascript/index.js

```diff
@@ -25,16 +27,123 @@
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
   }, (error) => console.log(error));
 
-
+*/
 
 // Iteration 1 - using callbacks
-// ...
+getInstruction("mashedPotatoes", 0, (step0) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step0}</li>`;
+
+  getInstruction("mashedPotatoes", 1, (step1) => {
+    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
+
+    getInstruction("mashedPotatoes", 2, (step2) => {
+      document.querySelector(
+        "#mashedPotatoes"
+      ).innerHTML += `<li>${step2}/</li>`;
+
+      getInstruction("mashedPotatoes", 3, (step3) => {
+        document.querySelector(
+          "#mashedPotatoes"
+        ).innerHTML += `<li>${step3}</li>`;
+
+        getInstruction("mashedPotatoes", 4, (step4) => {
+          document.querySelector(
+            "#mashedPotatoes"
+          ).innerHTML += `<li>${step4}</li>`;
+
+          document.querySelector(
+            "#mashedPotatoes"
+          ).innerHTML += `<li>Mashed potatos are ready!</li>`;
+          document
+            .querySelector("#mashedPotatoesImg")
+            .removeAttribute("hidden");
+        });
+      });
+    });
+  });
+});
 
 // Iteration 2 - using promises
-// ...
+obtainInstruction("steak", 0).then((step0) => {
```

Comment:
Paulina, here, you used the `.then()` method, however, you also chained your promises, and that's not the way we wanted.
Here's another approach that you could've used:
```
// Iteration 2 - using promises
obtainInstruction('steak', 0)
  .then((step0) => {
    document.querySelector('#steak').innerHTML += `<li>${step0}</li>`;
    return obtainInstruction('steak', 1);
  })
  .then((step1) => {
    document.querySelector('#steak').innerHTML += `<li>${step1}</li>`;
    return obtainInstruction('steak', 2);
  })
  .then((step2) => {
    document.querySelector('#steak').innerHTML += `<li>${step2}</li>`;
    return obtainInstruction('steak', 3);
  })
  .then((step3) => {
    document.querySelector('#steak').innerHTML += `<li>${step3}</li>`;
    return obtainInstruction('steak', 4);
  })
  .then((step4) => {
    document.querySelector('#steak').innerHTML += `<li>${step4}</li>`;
    return obtainInstruction('steak', 5);
  })
  .then((step5) => {
    document.querySelector('#steak').innerHTML += `<li>${step5}</li>`;
    return obtainInstruction('steak', 6);
  })
  .then((step6) => {
    document.querySelector('#steak').innerHTML += `<li>${step6}</li>`;
    return obtainInstruction('steak', 7);
  })
  .then((step7) => {
    document.querySelector('#steak').innerHTML += `<li>${step7}</li>`;
    document.querySelector('#steak').innerHTML += `<li>Steak is ready!</li>`;
    // Bonus 1
    document.querySelector('#steakImg').removeAttribute('hidden');
  });
```